engines/cine/detection.cpp

```diff
@@ -49,18 +48,11 @@ Common::Platform CineEngine::getPlatform() const { return _gameDescription->desc
 } // End of namespace Cine
 
 static const PlainGameDescriptor cineGames[] = {
-	{"cine", "Cinematique evo.1 engine game"},
 	{"fw", "Future Wars"},
 	{"os", "Operation Stealth"},
 	{0, 0}
 };
 
-static const Engines::ObsoleteGameID obsoleteGameIDsTable[] = {
```

Comment:
Why? Will it still work?

Reply:
I have removed the obsolete engine id tables in the cases where the old obsolete gameIds are the new migrated game ids. In these cases the engineId will be found by the new target upgrade code using the gameId.